packages/commonwealth/server/scripts/emailDigest.ts

```diff
@@ -1,11 +1,15 @@
+import { HotShotsStats } from '@hicommonwealth/adapters/build/index';
+import { stats } from '@hicommonwealth/core/build/index';
 import { models } from '@hicommonwealth/model';
 import { DynamicTemplate } from '@hicommonwealth/shared';
+import * as dotenv from 'dotenv';
```

Comment:
node 20 supports loading env files... should we start deprecating dotenv?

Reply:
I think that requires modifying the start-up command to supply a dotenv file. Dotenv is a little more programmatic since we can call it from anywhere for any number of `.env` files. Unless there we can reduce complexity greatly (which I don't think is the case), I don't see why we would use the built-in `.env` support just yet.